Specs/Number/Hindi/NumberRangeModel.json

```diff
@@ -211,43 +198,41 @@
     ]
   },
   {
-    "Input": "संख्या एक चौथाई स अधिक और आधे से कम है.",
+    "Input": "संख्या एक चौथाई से अधिक और आधे से कम है.",
     "NotSupported": "javascript, python",
-    "NotSupported": "dotnet",
     "NotSupportedByDesign": "javascript,python,java",
     "Results": [
       {
-        "Text": "एक चौथाई स अधिक और आधे से कम",
+        "Text": "एक चौथाई से अधिक और आधे से कम",
         "TypeName": "numberrange",
         "Resolution": {
           "value": "(0.25,0.5)"
         },
         "Start": 7,
-        "End": 34
+        "End": 35
       }
     ]
   },
   {
-    "Input": "यह संख्या तीन हजार नौ सौ पैंसठ से बड़ी है।",
+    "Input": "यह संख्या तीन हजार नौ सौ पैंसठ से बड़ी या उसके समान है।",
+    "Comment": "Retranslated due to incorrect translation.",
```

Comment:
Description of changes should happen at the PR level. The "Comment" field in the specs is specific for comments related to the spec itself, for example, some case not currently supported and the reason.

Reply:
ok, removed comments